tests/conftest.py

```diff
@@ -899,9 +896,8 @@ def one_variant_customannotation(request, customannotation_snv_file):
     LOG.info("Return one parsed variant with custom annotations")
     variant_parser = VCF(customannotation_snv_file)
 
-    for variant in variant_parser:
-        break
-
+    variant_iterator = iter(variant_parser)
```

Comment:
I think you can skip this and do `next(variant_parser)` directly as `cyvcf2.VCF` is a iterator

Reply:
Ah, I'll try that!